frame/bonded-finance/src/utils.rs

```diff
@@ -0,0 +1,3 @@
+#[cfg(any(feature = "runtime-benchmarks", test))]
+
+pub const MIN_VESTED_TRANSFER: u32 = 1_000_000;
```

Comment:
Created a new module that shares common information for "pallet" and "benchmark" usage. Although minimal, it also serves as a place for future additions.